internal/jet/expression.go

```diff
@@ -278,7 +278,7 @@ type customExpression struct {
 	parts []Serializer
 }
 
-func newCustomExpression(parts ...Serializer) Expression {
+func NewCustomExpression(parts ...Serializer) Expression {
```

Comment:
Without `New`. Just `CustomExpression` is fine.

Reply:
applied